compose/app/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,8 @@
+<resources>
+    <string name="app_name">ChartboostMediationDemo</string>
+    <string name="app_id">5a4e797538a5f00cf60738d6</string>
```

Comment:
We may want to see the app id that was used before. Although, we may want to double-check if we want to switch it.